docs/source/previous-versions.mdx

```diff
@@ -14,6 +14,8 @@ Both previous major versions of Apollo Server are currently [deprecated](/resour
 
 Deprecated versions continue to receive security patches and updates to address major regressions until their end-of-life date. They typically do not receive new features and other kinds of bug fixes may or may not be backported to deprecated versions at Apollo's discretion.
 
+End-of-life versions will remain available in the npm registry. They can still be installed and used in your projects, but Apollo does not guarantee any further support for them nor does Apollo recommend their use.
```

Comment:
maybe something about how other systems might change in ways that they don't work any more, like Studio may get changed in ways that Studio integration features from EOL versions don't work?

Reply:
took a stab at this: 

```EOL versions can still be used in your projects, but may no longer function as expected when used with Apollo Studio or GraphOS beyond their end-of-life date. Apollo does not guarantee support for or recommend the use of EOL versions```  

feel free to use or discard any bits that work

Reply:
I reworked it before I saw your comment. I should mention GraphOS but let me know what you think of the update @maryelizbeth